piccolo/apps/user/commands/list.py

```diff
@@ -0,0 +1,26 @@
+from piccolo.apps.user.tables import BaseUser
+
+
+def list():
+    """
+    List existing users.
+    """
+    users = BaseUser.select(
+        BaseUser.all_columns(exclude=[BaseUser.password])
+    ).run_sync()
+
+    if len(users) < 1:
+        return
+
+    ref_order = [column for column in users[0]]
+    width = {column: len(column) for column in ref_order}
+
+    for user in users:
+        for column in ref_order:
+            if len(str(user[column])) > width[column]:
+                width[column] = len(str(user[column]))
+
+    format_string = "|".join([f"{{:<{width[w]}}}" for w in ref_order])
+    print(format_string.format(*[w for w in ref_order]))
+    for user in users:
+        print(format_string.format(*[str(user[w]) for w in ref_order]))
```

Comment:
I think what you've done looks cool.

<img width="676" alt="Screenshot 2024-01-22 at 15 34 00" src="https://github.com/piccolo-orm/piccolo/assets/350976/f7fd0e42-dcfa-4009-a80c-8b429916159b">

What do you think about adding a space either side of the `|` symbols? Only if it doesn't add loads of complexity.

Reply:
There's a file called `piccolo/utils/printing.py` which has utilities for this kind of thing. We don't have anything which does exactly what you're doing, but maybe it's worth add it there so it's reusable?

Reply:
Done both things.